### ISSUE TITLE
Holy shit

### ISSUE THREAD

**Issue by bengdryan** [gjacksonotago/CalendarApp#30]
I can't believe this still exists... well I can but... *cringe*

**Comment by gjacksonotago** [comment]
Neither can I, I'd completely forgotten this was a thing.....

**Comment by bengdryan** [comment]
Fortunately we use Bit Bucket at my job. They'll never know!

**Comment by gjacksonotago** [comment]
Yeah we don't use anything like this at my work either! Our shame is hidden! We are saved.
